CONTRIBUTING.md

```diff
@@ -61,6 +61,7 @@ Supported filename extensions are:
 
 - ``.feature``: Signifying a new feature in Element Android or in the Matrix SDK.
 - ``.bugfix``: Signifying a bug fix.
+- ``.dev``: Signifying a change which is not ready to be user facing, typically a component of a larger feature.
```

Comment:
I'm not sure if `.dev` is the best extension might be a little too generic, what do people think?

Reply:
Maybe `.wip` since it's just an internal choose?

Reply:
sounds good to me :+1: